packages/plugins/plugin-sheet/src/graph/compute-node.ts

```diff
@@ -0,0 +1,52 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import { DetailedCellError } from 'hyperformula';
+
+import { Event } from '@dxos/async';
+import { Resource } from '@dxos/context';
+
+import { type ComputeGraph } from './compute-graph';
+import { type CellAddress } from '../defs';
+import { type CellScalarValue } from '../types';
+
+/**
+ * Individual "sheet" (typically corresponds to an ECHO object).
+ */
+// TODO(burdon): Factor out common HF wrapper from from SheetModel.
+export class ComputeNode extends Resource {
+  // TODO(burdon): Chaing events.
+  public readonly update = new Event();
+
+  constructor(
+    private readonly _graph: ComputeGraph,
+    public readonly sheetId: number,
+  ) {
+    super();
+  }
+
+  // TODO(burdon): Remove?
+  get graph() {
+    return this._graph;
+  }
+
+  get hf() {
+    return this._graph.hf;
+  }
+
+  getValue(cell: CellAddress): CellScalarValue {
```

Comment:
Is there a way to support other addressing formats for non-sheet objects? For example acessing table data

Reply:
you just need to be able to map to a cell address

Reply:
no, for tables, etc. we'd just use ECHO queries, no?